includes/polling/os/slxos.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
Could you please move this file to the new format, see https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery for more info, sorry for the inconvenience